components/measurement/CommonDetails.js

```diff
@@ -80,15 +80,12 @@ const CommonDetails = ({
             <Heading h={4}>Raw Measurement Data</Heading>
           </Box>
           <Box >
-            <Button
-              onClick={() => (
-                jsFileDownload(
-                  JSON.stringify(measurement, null, 2),
-                  'ooni-measurement-' + report_id + '.json')
-              )}
-              fontSize={11}
-              mx={3}
-              px={3}>Download JSON</Button>
+            <a href={measurementURL} download={downloadFilename}>
```

Comment:
This seems to work as a download link only when relative URLs are used. "Save Link As..." works but doesn't pick up the `downloadFilename` value.

Reply:
I am going to check if some backend change is necessary in order to make it download instead of showing it in the browser. However I also don't mind it appearing in the browser and prefer that to a client-side JS solution.

Reply:
I filed this API ticket for this: https://github.com/ooni/api/issues/68

Reply:
This is implemented in: https://github.com/ooni/api/pull/69